app/src/main/java/com/sun/music61/screen/play/PlayFragment.java

```diff
@@ -51,13 +55,22 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         mTrack = Objects.requireNonNull(getArguments()).getParcelable(ARGUMENT_TRACK);
         View rootView = inflater.inflate(R.layout.play_fragment, container, false);
         initView(rootView);
+        onListenerEvent();
         return rootView;
     }
 
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
         fetchDataTrack(mTrack);
+        // Testing
+        onTestPlayMusic();
+    }
+
+    private void onTestPlayMusic() {
+        mPlayerManager = new MediaPlayerManager(getContext());
```

Comment:
```suggestion
        mPlayerManager = MediaPlayerManager.getInstance(getContext());
```
Neu test thi k can push len dau